template/config/functions/bootstrap.js

```diff
@@ -181,6 +181,7 @@ async function importSeedData() {
   await setPublicPermissions({
     global: ["find"],
     page: ["find", "findone"],
+    'lead-form-submissions': ["create"]
```

Comment:
```suggestion
    'lead-form-submissions': ["create"],
```

Good catch! I'd just add a trailing comma for consistency